createAccountJava/src/main/java/com/brandon/createaccountjava/viewmodel/LegacyCreateAccountViewModel.java

```diff
@@ -0,0 +1,97 @@
+package com.brandon.createaccountjava.viewmodel;
+
+
+import static com.brandon.corejava.utilities.ConnectionErrors.NO_CONNECTION_ERROR;
+import static com.brandon.corejava.utilities.UserInputErrors.NO_INPUT_ERROR;
+import static com.brandon.createaccountjava.viewmodel.CreateAccountUiStates.IDLE;
+
+import android.net.ConnectivityManager;
+
+import com.brandon.createaccountjava.domain.CreateUserUseCaseJava;
+
+import javax.inject.Inject;
+
+import dagger.hilt.android.lifecycle.HiltViewModel;
+import io.reactivex.rxjava3.annotations.NonNull;
+import io.reactivex.rxjava3.core.BackpressureStrategy;
+import io.reactivex.rxjava3.core.Flowable;
+import io.reactivex.rxjava3.subjects.BehaviorSubject;
+
+@HiltViewModel
+public class LegacyCreateAccountViewModel extends CreateAccountViewModel {
+    private final CreateUserUseCaseJava createUserUseCaseJava;
+    private final ConnectivityManager connectivityManager;
+
+    private final BehaviorSubject<CreateAccountUiStateJava> uiState = BehaviorSubject.createDefault(
+            new CreateAccountUiStateJava(
```

Comment:
Make another constructor that defaults all these

Reply:
made a default